File: asp.net/WEB/App_Code/as2DotNet/IParamEncrypt.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

/// <summary>
///IParamEncrypt 的摘要说明
/// </summary>
public interface IParamEncrypt
{
    /**
     * 加密
     * @param param
     * @return
     */
    byte[] encrypt(byte[] param);

    /**
     * 解密
     * @param param
     * @return
     */
    byte[] decrypt(byte[] param);
}

File: java/javaSpring2.5/examples/TestProject/src/cn/as4j/bean/IParamEncrypt.java
package cn.as4j.bean;


/**
 * 参数加密接口
 * @author 破晓
 *
 */
public interface IParamEncrypt {

	
	/**
	 * 加密
	 * @param param
	 * @return
	 */
	public byte[] encrypt(byte[] param);
	
	/**
	 * 解密
	 * @param param
	 * @return
	 */
	public byte[] decrypt(byte[] param);
}
